src/vm/vm_core.rs

```diff
@@ -2158,77 +2248,6 @@ mod tests {
         );
     }
 
-    #[test]
```

Comment:
I deleted these tests because they belong to this part of the code  
https://github.com/lambdaclass/cairo-rs/issues/390